samples/snippets/fsharp/azure/blob-storage.fsx

```diff
@@ -172,32 +135,34 @@ ListBlobsSegmentedInFlatListing container |> Async.RunSynchronously
 //
 
 // Get a reference to a container.
-let appendContainer = blobClient.GetContainerReference("my-append-blobs")
+let appendContainer = service.GetBlobContainerClient("my-append-blobs")
 
 // Create the container if it does not already exist.
 appendContainer.CreateIfNotExists() |> ignore
 
 // Get a reference to an append blob.
-let appendBlob = appendContainer.GetAppendBlobReference("append-blob.log")
+let appendBlob = appendContainer.GetAppendBlobClient("append-blob.log")
 
 // Create the append blob. Note that if the blob already exists, the 
 // CreateOrReplace() method will overwrite it. You can check whether the 
 // blob exists to avoid overwriting it by using CloudAppendBlob.Exists().
-appendBlob.CreateOrReplace()
+appendBlob.CreateIfNotExists()
 
 let numBlocks = 10
 
 // Generate an array of random bytes.
 let rnd = new Random()
```

Comment:
```suggestion
let rnd = Random()
```

I believe that it is idiomatic to omit the `new` here since `Random` is not disposable.